problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -195,9 +205,18 @@ function MentoringWithStepsBlock(runtime, element) {
             showActiveStep();
             validateXBlock();
             updateNextLabel();
-            nextDOM.attr('disabled', 'disabled');
+            var step = steps[activeStep];
+            if (step.hasQuestion()) {
```

Comment:
@Kelketek these two new blocks are almost identical - would be good to refactor them into method.